pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -58,17 +58,22 @@ var (
 	noScheduleTaints      = []v1.Taint{{Key: "dedicated", Value: "user1", Effect: "NoSchedule"}}
 )
 
+func nowPointer() *metav1.Time {
```

Comment:
I'm ok with that :)
Two lines to get "now" pointer seems boring :) ; just code style, no block comments to this PR :).